Roles/Player/README.md

```diff
@@ -23,6 +23,7 @@
 * [@flaxscrip](https://flaxscrip.github.io/flaxscrip-space/)
 * [@aiggdev](https://github.com/aiggdev/aiggdev-space/)
 * [@angelabacca](https://angelabacca.github.io/angela-player-space/)
+* [@foundationx](https://github.com/foundationx)
```

Comment:
Is this a placeholder until @foundationx creates a player space?

Reply:
Yes I will be creating a player space

Reply:
I was walking @foundationx through the Nomicon process. 

@foundationx - What David is truly saying is that I went out-of-process and should only submit the Pull Request once we've had proper discussion (in the issue). Then we make the change to the Nomicon and submit the PR for vote.

@macterra - Yes - I went ahead in my demo to show the PR step.

Reply:
@Flaxscrip okay here is the player URL https://foundationx.github.io/foundationx-space/